dev-ops/common/actions/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#DESCRIPTION: Update Shopware after deployment
+
+INCLUDE: ./cache.sh
+
+bin/console database:refresh-migration
+
+bin/console plugin:refresh
+
+bin/console database:migrate --all
+bin/console database:migrate-destructive --all
```

Comment:
You cannot rollback if you execute the destructive migrations.

Reply:
So the destructive migrations are to be executed manually on purpose as soon as you know that everything is fine just to clean things up?